ci/Containerfile

```diff
@@ -40,6 +41,13 @@ RUN dnf update -y && \
     dnf autoremove -y && \
     dnf clean all
 
+# There's no simply way to install a automation-library version that's
+# consistent with other environments via use of the ../lib.sh specification.
+# Just use the "latest" version and assume it will work for this containers context.
+RUN url='https://github.com/containers/automation/releases/latest/download/install_automation.sh' && \
```

Comment:
Why is this different from the URL in `lib.sh`? https://github.com//containers/automation_images/blob/841849cd3e9e301f202ab886f5cc658d9a21dc98/lib.sh#L57

And, did you test it? (Is it possible to test?) I don't understand Containerfile syntax enough to know if this `var=xxx && so on` syntax will work.

And, it really disturbs me to have this sort of moving-target usage... but since it's already being done in lib.sh, I guess we live with it.

Reply:
Thanks for fixing it. Final (I think) question: since one of these is in a Containerfile, and the other in a .sh file, is there any way that it could be refactored so the duplication is eliminated? (I couldn't figure out where `ci/Containerfile` gets run, but maybe there's a way for the caller to set the URL via envariable?)

Reply:
> maybe there's a way for the caller to set the URL via envariable?

Actually that gives me an idea, I think this might be possible...

Reply:
...force-pushed something that might work.

Reply:
[Yep it works!](https://cirrus-ci.com/task/5036839777075200?logs=build#L1123)